borgmatic/config/validate.py

```diff
@@ -59,6 +55,17 @@ def parse_configuration(config_filename, schema_filename):
     return parsed_result
 
 
+def clean_pykwalify(schema):
+    '''
+    pykwalify gets angry if the example field is not a string. So rather than bend to its will,
+    simply remove all examples before passing the schema to pykwalify.
+    '''
+    for section_name, section_schema in schema['map'].items():
+        section_schema.pop('example', None)
+        if 'map' in section_schema:
+            clean_pykwalify(section_schema)
```

Comment:
I recommend adding tests for this function. And perhaps mocking it out of any unit tests of its direct caller (`parse_configuration()`).

Alternately, if you lose some of the schema nesting as suggested above, you may not actually need this change right now.

Reply:
Alright, if not needed by the schema, I also suggest not to change this for now. If needed at a later point in time, it could still be (re-)introduced.

Reply:
Okay, makes sense.